tests/assets/dynamic-addon-cards.test.js

```diff
@@ -72,6 +72,24 @@ describe(__filename, () => {
       return updateAddonCard(card, { parsedUserAgent });
     };
 
+    const expectDisabledInstallButton = ({ getFirefoxButton, downloadURL }) => {
+      const button = getFirefoxButton.querySelector('.GetFirefoxButton-button');
+
+      expect(button.classList).toContain('Button--disabled');
+      expect(button.getAttribute('aria-disabled')).toEqual('true');
+      expect(button.getAttribute('href')).toEqual(downloadURL);
+
+      const event = new window.MouseEvent('click');
+      const preventDefaultSpy = jest.spyOn(event, 'preventDefault');
+      button.dispatchEvent(event);
+      expect(preventDefaultSpy).toHaveBeenCalled();
+    };
+
+    const expectEnabledInstallButton = ({ getFirefoxButton }) => {
+      const button = getFirefoxButton.querySelector('.GetFirefoxButton-button');
+      expect(button.classList).not.toContain('Button--disabled');
+    };
```

Comment:
Do we need to verify here that `preventDefault` is *not* triggered when the button is clicked?

Reply:
I don't know if we _need_ to but I added an assertion anyway.